### ISSUE TITLE
SQLITE_OK, SQLITE_DENY and SQLITE_IGNORE not explicitly documented

### ISSUE THREAD

**Issue by CAM-Gerlach** [python/cpython#96096]
After #96095 is merged, the only remaining broken references/warnings in the `sqlite3` library documentation are for the `SQLITE_OK`, `SQLITE_DENY` and `SQLITE_IGNORE` module-level constants that are referenced with `:const:` under `Connection.set_authorizer`, but not explicitly documented anywhere (as with the similar reference to `sqlite.PrepareProtocol` fixed in #94321). Should we at least add brief mentions of them to module constants, e.g.

```rst
.. data:: SQLITE_OK
          SQLITE_DENY
          SQLITE_IGNORE

   Flag that *authorizer_callback* of :meth:`Connection.set_authorizer`
   should return to indicate whether:

   * Access is allowed (:const:`!SQLITE_OK`),
   * The SQL statement should be aborted with an error (:const:`!SQLITE_DENY`)
   * The column should be treated as a ``NULL`` value (:const:`!SQLITE_IGNORE`)
```

Also, maybe `Connection.set_authorizer()`'s docs should be updated to say something like "The callback should return :const:`SQLITE_OK`. :const:`SQLITE_DENY`. or :const:`SQLITE_IGNORE` to control the resulting behavior for the column" rather than repeating ourselves.

@erlend-aasland what do you think?

**Comment by erlend-aasland** [comment]
Yep, we should document these, as they are part of the authorizer interface. Note that only `SQLITE_DENY` and `SQLITE_IGNORE` are specific to the authorizer callback.

FYI: Every SQLite result code is available as module level attributes. There are 152 of them. It may not make sense to cover each and everyone. Anyway, that is out of scope of this issue.

**Comment by CAM-Gerlach** [comment]
Thanks @erlend-aasland . As the resident `sqlite3` expert, I would ideally defer to your expertise, but if you would rather I do so, I could submit a PR with the above changes—up to you.

**Comment by erlend-aasland** [comment]
I won't have time the next few days, so if you feel the urge, please go ahead 🙂

**Comment by CAM-Gerlach** [comment]
Thanks, opened as #96134
